app/serializers/prison_transfer_reason_serializer.rb

```diff
@@ -3,7 +3,9 @@
 class PrisonTransferReasonSerializer
   include JSONAPI::Serializer
 
+  INCLUDED_ATTRIBUTES = %i[title key disabled_at].freeze
```

Comment:
Aren't these sparse fields in json:api speak?

Reply:
Interesting question. We do not support the `fields?` parameter in the API at the moment. Reading up on [spare fieldsets](https://jsonapi.org/format/#fetching-sparse-fieldsets) I think it is fine to give a subset of fields:
> ...If a client does not specify the set of fields for a given resource type, the server MAY send all fields, a subset of fields, or no fields for that resource type.

Reply:
I mean the name is obvious enough 👍